module-sys/common/include/system/Common.hpp

```diff
@@ -128,6 +129,8 @@ inline const char *c_str(sys::BusChannel channel)
         return "BluetoothNotifications";
     case sys::BusChannel::PhoneLockChanges:
         return "PhoneLockChanges";
+    case sys::BusChannel::LayoutChangeRequests:
```

Comment:
why not action? and why in common code when it's only for bell?

Reply:
Its my first case of app<->app communication and I got suggested to use channels mechanism. Shall I rewrite that part to use actions?

Reply:
reworked to action